[DOCS] Add memory limit details in update job API (#74517)
elastic/elasticsearch: docs/reference/ml/anomaly-detection/apis/update-job.asciidoc

include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-memory-limit-ad]
+
--
[NOTE]
=======
* You cannot decrease the `model_memory_limit` value below the current usage. To

> determine the current usage, refer to the `model_bytes_memory_limit` value in

Reviewer: ```suggestion
determine the current usage, refer to the `model_bytes` value in
```

> the <<ml-get-snapshot,get model snapshots>> or <<ml-get-job-stats,get job stats>> 
> APIs.

Reviewer: ```suggestion
the <<ml-get-job-stats,get job stats>> API.
```

* If the `memory_status` property in the

> <<ml-get-snapshot-results,`model_size_stats` object>> has a value of 

Reviewer: ```suggestion
<<modelsizestats,`model_size_stats` object>> has a value of 
```
